recipes/joypy/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set name = "joypy" %}
+{% set version = "0.1.10" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 0deca06dd7c36a1e1a4d545e461f3cb39b963bc184df69948fb32faff753606d
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:    
+    - pip
+    - python    
+  run:
+    - matplotlib
+    - {{ pin_compatible('numpy') }}
```

There is no need for the `pin_compatible` as this isn't compiling anything:
```suggestion
    - numpy
```